test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/Semantics/CsdlSemanticsEntityContainerTests.cs

```diff
@@ -70,5 +70,35 @@ public void EnsureActionImportActionPropertyIsUnresolvedAction()
             Assert.Equal(EdmContainerElementKind.ActionImport, imports[0].ContainerElementKind);
             Assert.Null(imports[0].EntitySet);
         }
+
+        [Fact]
+        public void NavigationPropertyBindingsReturned()
+        {
+            // arrange
+            var entitySet1 = new CsdlEntitySet("EntitySet1", "unknown", new[] { 
+                new CsdlNavigationPropertyBinding("foo", "bar", testLocation)
+            }, testLocation);
+            var singleton1 = new CsdlSingleton("Singleton", "unknown", new[] {
+                new CsdlNavigationPropertyBinding("foo", "bar", testLocation)
+            }, testLocation);
+            var csdlEntityContainer = CsdlBuilder.EntityContainer("Container", entitySets: new [] { entitySet1 }, singletons: new[] { singleton1 } );
+            var schema = CsdlBuilder.Schema("FQ.NS", csdlEntityContainers: new CsdlEntityContainer[] { csdlEntityContainer });
+            var csdlModel = new CsdlModel();
+            csdlModel.AddSchema(schema);
+            
+            // act
+            var container = new CsdlSemanticsEntityContainer(
+                new CsdlSemanticsSchema(
+                    new CsdlSemanticsModel(
+                        csdlModel,
+                        new EdmDirectValueAnnotationsManager(),
+                        Enumerable.Empty<IEdmModel>()),
+                    schema),
+                csdlEntityContainer);
+            var bindings = container.GetNavigationPropertyBindings().ToList();
+
+            // assert
+            Assert.Equal(2, bindings.Count);

```

Comment:
I think we should also assert the actual bindings are what's expected?

Reply:
@habbes Added requested asserts